src/templates/partials/hiring_footer.hbs

```diff
@@ -1,7 +1,7 @@
 {{#markdown}}
 
 <p style="border-top: dotted 1px #CCC; margin-top: 40px; padding-top: 20px">
-Are you passionate about JavaScript, modeling and the web?
+Are you passionate about JavaScript, modeling, and the web?
```

Comment:
I don't quite get the call to action:

> Join Camunda and build modeling tools people <3

What do you build when you build modeling tools people? 🤔